frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/os-flavor.tsx

```diff
@@ -184,21 +186,34 @@ export const OSFlavor: React.FC<OSFlavorProps> = React.memo(
               onChange={(v) => onChange(VMSettingsField.CLONE_COMMON_BASE_DISK_IMAGE, v)}
             />
           </FormField>
-          {cloneBaseDiskImage && (
-            <Text>
-              View the cloned disk in the{' '}
-              <Button
-                isDisabled={!goToStorageStep}
-                isInline
-                onClick={goToStorageStep}
-                variant={ButtonVariant.link}
-              >
-                <strong>storage</strong>
-              </Button>{' '}
-              step
-            </Text>
-          )}
         </FormFieldRow>
+        <FormFieldRow
+          field={mountWindowsGuestToolsField}
+          fieldType={FormFieldType.INLINE_CHECKBOX}
+          loadingResources={loadingResources}
+        >
+          <FormField>
+            <Checkbox
+              className="kv-create-vm__input-checkbox"
+              id={getFieldId(VMSettingsField.MOUNT_WINDOWS_GUEST_TOOLS)}
+              onChange={(v) => onChange(VMSettingsField.MOUNT_WINDOWS_GUEST_TOOLS, v)}
+            />
+          </FormField>
+        </FormFieldRow>
+        {(cloneBaseDiskImage || mountWindowsGuestTools) && (
+          <Text className="kv-create-vm__input-checkbox">
```

Comment:
i know it works, but the name does not reflect the reality :)

either
- make more generic name
- make 2nd class

Reply:
:-) yea

Reply:
changed to `kv-create-vm__input-text-help-msg` :+1:

Reply:
any reason you can't use `pf-c-form__helper-text` here instead?